spec/index.html

```diff
@@ -2037,7 +2037,7 @@ <h3>Algorithm</h3>
                 <li id="hndq-5-4-4">For each <var>related</var> in <var>p</var>:
                   <ol>
                     <li id="hndq-5-4-4-1">If a canonical identifier has been issued for
-                      <var>related</var> by <var>issuer</var>, append the string <code>_:</code>, followed by 
+                      <var>related</var> by <a>canonical issuer</a>, append the string <code>_:</code>, followed by 
                       <var>related</var>, to <var>path</var>.</li>
```

Comment:
```suggestion
                      <var>related</var> by <a>canonical issuer</a>, append the string <code>_:</code>, followed by 
                      the canonical identifier for <var>related</var>, to <var>path</var>.</li>
```

(The original text suggests that the value of _related_ should be appended to _path_...)

Reply:
Yes to this.

Reply:
I can't apply both of these, so I'll merge this change into the previous suggestion.